src/components/QuickOpenModal.js

```diff
@@ -170,11 +170,11 @@ export class QuickOpenModal extends Component<Props, State> {
     }
 
     if (searchType === "variables") {
-      selectSource(selectedSource.get("id"), {
-        location: {
-          ...(item.location != null ? { line: item.location.start.line } : {})
-        }
-      });
+      const selectResultLocation = { sourceId: selectedSource.get("id") };
+      if (item.location != null) {
```

Comment:
This might be rewritten in a nicer way similar to: 
` ...(!isNaN(columnNumber) ? { column: columnNumber } : null)`